doc/src/DebugAndLogging.md

```diff
@@ -137,10 +137,10 @@ The `ROCBLAS_LAYER` environmental variable can be set to output GEMM information
 Additionally, using environment variable "MIOPEN_GEMM_ENFORCE_BACKEND", can override the default behavior. The default behavior which is to use
 both MIOpenGEMM and rocBlas depending on the input configuration:
 
-* `MIOPEN_GEMM_ENFORCE_BACKEND=0`, use MIOpenTensile for FP32, use rocBLAS for FP16 if enabled
-* `MIOPEN_GEMM_ENFORCE_BACKEND=1`, use rocBLAS if enabled
-* `MIOPEN_GEMM_ENFORCE_BACKEND=2`, use MIOpenGEMM for FP32, use rocBLAS for FP16 if enabled
-* `MIOPEN_GEMM_ENFORCE_BACKEND=3`, no gemm will be called
+* `MIOPEN_GEMM_ENFORCE_BACKEND=1`, use MIOpenTensile for FP32, use rocBLAS for FP16 if enabled
```

Comment:
Please maintain backward compatibility for the sake of who is already using this.

Reply:
done